app/tests/test_activity_api.py

```diff
@@ -0,0 +1,51 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from core.models import Activity
+from api.serializers import ActivitySerializer
+
+from tests import create_new_user, create_new_project, create_new_activity
+
+
+class PrivateActivityTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user = create_new_user("Seba")
+        self.client.force_authenticate(self.user)
+        self.project = create_new_project(user=self.user, title="Learn Python")
+        self.activity = create_new_activity(project=self.project)
+        self.general_url = reverse('api:activity-list')
+        self.detail_url = reverse('api:activity-detail', args=[self.activity.id])
+        self.client.patch(self.detail_url) # Finish started activity
+
+    def test_activity_list_not_allowed(self):
+        """Test that retrieving list of activities is not allowed"""
+        activity1 = create_new_activity(project=self.project)
```

Comment:
no point in creating activity1 or activity2 if what you aiming for is to only get 405 error, it only makes test slower

Reply:
OK removed